src/View/Form/EntityContext.php

```diff
@@ -711,10 +711,7 @@ public function attributes(string $field): array
             return [];
         }
 
-        $column = (array)$table->getSchema()->getColumn(array_pop($parts));
-        $allowed = ['length' => null, 'precision' => null];
```

Comment:
I agree that having the comment could be useful. Why not add only that. One drawback to adding everything is that the interface for attributes gets muddy and hard to keep consistent.